packages/byods-demo-server/src/index.ts

```diff
@@ -0,0 +1,8 @@
+import logger from 'jet-logger';
+import server from './server';
+
+// **** Run **** //
+
+const SERVER_START_MSG = `Express server started on port: ${3000}`;
```

Comment:
Decided to decouple the `index.ts` and core server handling, felt it would be smoother.